File: documentation/user/en/query/ordering/examples/comparable/primary-key-natural.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		EntityLocaleEquals(CultureInfo.GetCultureInfo("cs"))
        	),
        	OrderBy(
        		EntityPrimaryKeyNatural(Desc)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("name")
        		)
        	)
        )
	)
);

File: documentation/user/en/query/ordering/examples/comparable/primary-key-natural.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					entityLocaleEquals(Locale.forLanguageTag("cs"))
				),
				orderBy(
					entityPrimaryKeyNatural(DESC)
				),
				require(
					entityFetch(
						attributeContent("name")
					)
				)
			)
		);
	}
);
